test/unit/throw.spec.js

```diff
@@ -13,11 +14,13 @@ var STATE_FAILED = Runnable.constants.STATE_FAILED;
 describe('a test that throws', function () {
   var suite;
   var runner;
+  var sandbox;
   var uncaughtHandlers;
 
   beforeEach(function () {
     suite = new Suite('Suite', 'root');
     runner = new Runner(suite);
+    sandbox = sinon.createSandbox();
```

Comment:
```suggestion
  var uncaughtHandlers;

  beforeEach(function () {
    suite = new Suite('Suite', 'root');
    runner = new Runner(suite);
```

No need to create a sandbox, sinon comes with a default one.

Though, should add:

https://github.com/mochajs/mocha/blob/2f3fedcc41cbb9d3e503d84098fcc07d7c3c49f1/test/unit/errors.spec.js#L8-L10